src/nvlink/wrapper_32bit.py

```
@@ -0,0 +1,587 @@
+                logger.info(
+                    "NV-Link initialized successfully",
+                    sid=self.sid,
+                    init_key=init_key if init_key != self.sid else None,
+                )
+            else:
+                if result == -100:
+                    error_msg = "地方競馬DATAのサービスキーが設定されていません。"
+                else:
+                    error_msg = "NV-Link initialization failed"
+                logger.error(error_msg, error_code=result, sid=self.sid, init_key=init_key)
+                raise NVLinkError(error_msg, error_code=result)
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)